projects/plugins/crm/includes/ZeroBSCRM.AJAX.php

```diff
@@ -5349,13 +5349,18 @@ function zeroBSCRM_AJAX_previewSegment() {
 
 			// return error str
 			$error_string = $exception->get_error_code();
+			$status       = 500;
+			if ( $error_string === 'segment_condition_produces_no_args' ) {
+				$status = 400;
+			}
```

Comment:
❤️ Probably could switch it to 500 all around, but I'm not familiar with fail cases and this works for now.